app/views/doc/ext.html.erb

```diff
@@ -88,7 +94,13 @@
           <p class='description'>
             Your Guide to deliver great code using TFS and Git.
           </p>
-        </li>	      
+        </li>
+        <li>
+          <h4><a href="https://try.github.io/">Git Tutorial - Try Git</a></h4>
```

Comment:
We already link to Try Git on the front page. It's probably reasonable to have a link here in the tutorial section, too, though.

Reply:
I never noticed that 🤔